src/validators/username.js

```diff
@@ -0,0 +1,14 @@
+import i18n from '@dhis2/d2-i18n'
+import { isEmpty, isString } from './helpers/index.js'
+
+const invalidUsernameMessage = i18n.t(
+    'Please provide a username between 1 and 255 characters'
+)
+
+const username = value =>
```

Comment:
Same problem as `password`

Reply:
I've renamed to dhis2Username in https://github.com/dhis2/ui-forms/pull/164/commits/c00ada596d1c1119e79ac5162204d4067a36dcc7